cost-analyzer/templates/cost-analyzer-smtp-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.kubecostProductConfigs }}
+{{- if .Values.kubecostProductConfigs.smtp }}
+{{- if .Values.kubecostProductConfigs.smtp.enabled }}
```

Comment:
These can be condensed into a single statement.

Reply:
I was following the example laid out [here](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/templates/cost-analyzer-pkey-configmap.yaml). How would you condense these conditionals? I'm not intimately familiar with template conditionals, yet.

Reply:
```
{{- if (((.Values.kubecostProductConfigs).smtp).enabled) }}
```
And remove two `{{- end -}}` statements from the bottom.